fatiando/gridder.py

```diff
@@ -233,6 +238,45 @@ def scatter(area, n, z=None, seed=None):
     return arrays
 
 
+def circular_scatter(area, n, random=False, seed=None):
+    """
+    Generate a set of n points positioned in a circular array.
+
+    The diameter of the circle is equal to the smallest dimension of the area
+
+    Parameters:
+
+    * area : list = [x1, x2, y1, y2]
+        Area inside of which the points are contained
+    * n : int
+        Number of points
+    * random : True or False
+        If True, positions of the points on the circle will be chosen at random
+    * seed : None or int
+        Seed used to generate the pseudo-random numbers if `random==True`.
+        If `None`, will use a different seed every time.
+        Use the same seed to generate the same random sequence.
+
+    Result:
+
+    * points : list
+        List of (x, y) coordinates of the points
```

Comment:
@victortxa could you please update the return description. You can copy it from `scatter`:

     * ``[x, y]``
         Numpy arrays with the x and y coordinates of the points